manifest.json

```
@@ -4691,6 +4691,58 @@
             }
           ]
         },
+        {
+          "productVersion": "$(dotnet|8.0|product-version)",
+          "sharedTags": {
+            "$(dotnet|8.0|product-version)-jammy-chiseled-composite": {},
+            "8.0-preview-jammy-chiseled-composite": {}
+          },
+          "platforms": [
+            {
+              "buildArgs": {
+                "REPO": "$(Repo:runtime)"
```

```suggestion
                "REPO": "$(Repo:runtime-deps)"
```

We don't want to base this on the runtime image, since the aspnet composite tarball contains its own runtime.

---

Same goes for all other new manifest entries.

---

Oops I must've missed it when duplicating the template. Thanks for addressing it Logan!